### DISCUSSION TITLE
Log Export and Import

### DISCUSSION THREAD

**Discussion by MRCondon** [N0BOY/FT8CN#37] [General]
I exported my log to adif to reload it into a new laptop... it worked but the "DIC" alert indicators did not load ;from the import.   They were all marked DIC even though they were worked... Any help or a patch???

**Comment by N0BOY** [comment]
Which version does this bug associated with? The newly released v0.88 P2? 

What logging software are you using on your PC?

Did the FT8CN exported ADIF file work before on the same logging software?

Please share those info to help debugging. Thanks.

**Comment by MRCondon** [comment]
I am still on .87, I just now saw the new patch.
I can retry on .88 ...How do I update by the way, do I need to uninstall the apk and install the new one ... Or?
The operational interface of the program is excellent!!


Thanks, having a load of fun.
Mike, NE4S

    On Tuesday, May 2, 2023 at 11:29:35 AM EDT, NØBOY ***@***.***> wrote:  
 
 


Which version does this bug associated with? The newly released v0.88 P2?

**Comment by MRCondon** [comment]
BTW, I looked for a version number on the .87 code I am running, and could not find one,.
I did find it in the settings/about page

You might als consider putting the Ver # on one/all of the pages title area.

I will google apk patches too...And, no hurries on anything for me, all in good time.
73

Mike

    On Tuesday, May 2, 2023 at 11:29:35 AM EDT, NØBOY ***@***.***> wrote:  
 
 


Which version does this bug associated with? The newly released v0.88 P2?

**Comment by MRCondon** [comment]
OK, I am a SLOW learner...In finding the Ver # in "About", I found the .88 patch update..."Get the latest version"
And I did the update.  

The Situation on the "DIC" alerts didn't change, all the old QSOs are in the log, but decodes do not reflect worked countries.
Thanks for your patience...
Mike

    On Tuesday, May 2, 2023 at 11:29:35 AM EDT, NØBOY ***@***.***> wrote:  
 
 


Which version does this bug associated with? The newly released v0.88 P2?

**Comment by N0BOY** [comment]
The app displays current version on the startup screen.

To update, you can download the newer version and override the older one. 

I will bring issue to the developer. Thanks for reporting the problem.

**Comment by N0BOY** [comment]
Are you loading the ADIF file back into FT8CN app? Or loading into other logging software?

If loading back to FT8CN app, you have to terminate the app and restart it. This way, it will load the DIC marks.

**Reply by bg7yoz** [reply]
是的，导入ADI文件后，FT8CN需要重新启动"DIC"才会有标记。
FT8CN在启动时，会查找曾经通联过的“DIC”，目前的版本，导入数据后没有做更新“DIC”处理，这个问题将在0.89解决。

**Comment by MRCondon** [comment]
GM,ADIF just downloaded to PC file.Phone turned offLaptop turned on, radio and FT8CN started.On PC go to laptop site for Log actions.Import the file taken from phone to laptop.
I did not load to or from another app.All the Q that decode had the Marks on.

I then ran the patch for .88 having stopped the .87 a couple times..88 still had all DIC marks on log entries...   
I can repeat.How do I delete the whole log in the laptop now, in order to reload the file?
Detail...  on a set of decodes, the D is always on, sometimes the C is off, less often the I is off.But always the D is on, and mostly the whole string DIC.
Thanks,Mike
 

    On Tuesday, May 2, 2023 at 09:43:16 PM EDT, NØBOY ***@***.***> wrote:  
 
 


Are you loading the ADIF file back into FT8CN app? Or loading into other logging software?

If loading back to FT8CN app, you have to terminate the app and restart it. This way, it will load the DIC marks.

**Comment by MRCondon** [comment]
謝謝，我會等。。Mike

    On Tuesday, May 2, 2023 at 09:57:45 PM EDT, bg7yoz ***@***.***> wrote:  
 
 


是的，导入ADI文件后，FT8CN需要重新启动"DIC"才会有标记。
FT8CN在启动时，会查找曾经通联过的“DIC”，目前的版本，导入数据后没有做更新“DIC”处理，这个问题将在0.89解决。
